go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/cilium/deepequal-gen v0.0.0-20231116094812-0d6c075c335f
 	github.com/cilium/dns v1.1.51-0.20231120140355-729345173dc3
 	github.com/cilium/ebpf v0.12.4-0.20240110182413-87531ae5ba83
+	github.com/cilium/endpointslice-controller v0.0.0-20240117201440-8a243bb50989
```

Comment:
@MrFreezeex 

I've submitted some changes into cilium/endpointslice-controller to change the way the logs are being logged. Now it checks for the "Kind" that we are using for the endpoint. We can also controller's name so that we know the controller of the endpoint slice that we are managing. Can you update the go mod to `github.com/cilium/endpointslice-controller 4c5454d11b8174da310855b34a8d81a73850086b`